frontend/src/components/TaskRelationTable.module.scss

```
@@ -3,7 +3,7 @@
 .listContainer {
   flex-grow: 1;
   min-width: 370px;
```

```suggestion
  min-width: 387px;
```

This should be increased to make room for the task name div, 387px seems to be enough.

The problem only appears when the text *just* fits to one line on 210px div, which is the defined width and used in measuring. But then the actual row has less room for the div, and so the text goes to two rows eating up the gap between the items.